src/app/clients/client-stepper/client-general-step/client-general-step.component.html

```diff
@@ -16,7 +16,7 @@
 
     <mat-form-field fxFlex="48%">
       <mat-label>Legal Form</mat-label>
-      <mat-select formControlName="legalFormId">
+      <mat-select required formControlName="legalFormId">
```

Comment:
Are you sure this is right? For non-entity client, the legal form cannot be required

Reply:
In both cases (Person and Entity) the Legal Form is required